src/object.d

```diff
@@ -1740,6 +1740,19 @@ class Throwable : Object
             }
         }
     }
+
+    /**
+     * Get the message describing the error.
+     * Base behavior is to return the `Throwable.msg` field.
+     * Override to return some other error message.
+     *
+     * Returns:
+     *  Error message
+     */
+    const(char)[] message() const @__future
```

Comment:
The way how it's implemented in DMD I think adding `@__future` to the right side is not valid. I'll double check.